assets/app/scripts/controllers/catalog/images.js

```diff
@@ -25,7 +25,6 @@ angular.module('openshiftConsole')
               imageRepoTag: imageRepoTag,
               name: imageRepo.metadata.name + ":" + imageRepoTag
             };
-            $scope.images.push(image);
```

Comment:
looks like we're inspecting `imageRepo.spec.tags[n].annotations.tags` for a "builder" tag... does that get echoed in status?